src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -132,6 +134,42 @@ trait IterableOps[+A] extends Any {
   }
 
   override def toString = s"$className(${mkString(", ")})"
+
+  def sameElements[B >: A](that: Iterable[B]): Boolean = {
+  val these = this.iterator()
+  val those = that.iterator()
+  while (these.hasNext && those.hasNext)
```

Comment:
Something is wonky about the indentation of this while loop. Also, any reason this shouldn't be a one-liner?  The extra lines just give an opportunity to get confused about indentation.